configure.ac

```diff
@@ -5149,7 +5149,7 @@ AC_CHECK_FUNCS([ \
   fork fork1 fpathconf fstatat ftime ftruncate futimens futimes futimesat \
   gai_strerror getegid geteuid getgid getgrent getgrgid getgrgid_r \
   getgrnam_r getgrouplist gethostname getitimer getloadavg getlogin \
-  getpeername getpgid getpid getppid getpriority _getpty \
+  getpeername getpgid getpid getppid getpriority _getpty getdtablesize \
```

Comment:
You need to run `make regen-configure`: https://docs.python.org/dev/using/configure.html#configure-script

Reply:
Hmm, my current system is Win, what should I do?

Reply:
Please give me some time, I can't seem to pull: docker: Error response from daemon: Get "https://registry-1.docker.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers).